### PR TITLE
Feature/youngjae/remove black image

### PR DESCRIPTION
**Pull request by byeongal**

### CODE REVIEW

File: src/bot.py [hunk 1]
```diff
@@ -156,10 +156,19 @@ async def call_back(interaction: discord.Interaction):
...
+                            if result[str(i + 1)]["is_filtered"]:
+                                button_list[i].callback = on_click_button(result[str(i + 1)]["origin_url"])
+                            else:
+                                button_list[i].callback = on_click_button(result[str(i + 1)]["url"])
                             view.add_item(button_list[i])
                         message_embed.set_image(url=result["grid"]["url"])
+                        if sum([each["is_filtered"] for each in result.values()]):
+                            warning_message_list.append("Potential NSFW content was detected in one or more images.")
+                            warning_message_list.append(
+                                "If you want to have the original image, press the button below."
```
Comment (589hero) [inline src/bot.py]: minor) 버튼을 누르면 이미지를 보여주는 거니까 `"If you want to have the original image, press the button below."`를 `"If you want to see the original image, press the button below."`로 바꾸는건 어떤가요?

**Review by 589hero — DISMISSED** [review]
LGTM with a minor comment

File: src/bot.py (hunk 1 above)
Comment (Laeyoung) [reply src/bot.py]: 좋네요!

**Review by 589hero — APPROVED** [review]
LGTM

**Pull request merged**